app/res/drawable/label_floating_bad.xml

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android" >
-    
-    <item>
-        <shape android:shape="rectangle"> 
-            <solid android:color="@color/solid_dark_red"/>
+     <item>
+        <shape android:shape="rectangle">
+            <solid android:color="@color/cc_attention_negative_color"/>
          </shape>
     </item>
-
+    <item android:left="8dp" android:right="8dp">
```

Comment:
These all have the same dimensions, right? would be good to style them somehow so these drawables are consistent without needing to keep them in sync